src/features/antispam.ts

```diff
@@ -30,7 +30,12 @@ export const onMessage: OnMessageHandler = async (client, message) => {
 
   const emojisCount = message.content.match(emojiRegex)?.length ?? 0;
 
-  if (emojisCount > MAX_EMOJI_COUNT) {
+  if (
+    emojisCount >
+    (message.channelId === '766393115044216854'
```

Comment:
To make the code clear for future readers this string could be extracted in a constant variable like `INTRODUCTIONS_CHANNEL_ID` similar to how it is done in the other files: https://github.com/vercel/nextjs-discord-bot/blob/dfe085c3edd4181a2a12643fa738133af838655f/src/features/countdown.ts#L11

Reply:
Thanks! Modified to do this